msteams-platform/tabs/how-to/personal-tab-navigation.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Personal tab app navigation
+description: Learn about the options for navigation within a personal tab app using the TeamsJS library.
+ms.date: 10/26/2022
+ms.topic: conceptual
+ms.custom: m365apps
+ms.localizationpriority: medium
+---
+
+# Navigate within a personal tab app
+
+The TeamsJS library provides support for navigation within a personal app. This article discusses the options available, which includes types of navigation such as that between tabs within the app or through use of Teams UI components like the back button.
+
+[!INCLUDE [sdk-include](~/includes/sdk-include.md)]
```

Comment:
Would you mind updating the include to **2.x.x** as part of this PR? Thanks!

Reply:
Sure thing. Updated the include to say 2.x.x. Will be in next commit shortly